src/index.js

```diff
@@ -28,9 +28,9 @@ const is = require('is');
 const PKG = require('../package.json');
 const v1 = require('./v1');
 
-const Snapshot = require('./snapshot.js');
-const Subscription = require('./subscription.js');
-const Topic = require('./topic.js');
+const Snapshot = require('./snapshot');
```

Comment:
<insert grump about unrelated changes in a commit here>

Reply:
apologies ❤️